navigating-new-codebases/navigating-new-codebases.md

```diff
@@ -18,6 +18,7 @@ N/A
 
 - Companies such as GitHub and Airbnb uses Ruby on Rails whereas Instagram, Dropbox, and Spotify use Django. Most, if not all, of the source code, has already been written - as a web developer, you will need to quickly interpret the code and either add to the code or debug any issues associated with the code.
 
+
```

Comment:
Mentors: no points for this white space

Reply:
@alodahl sorry but couldn't get it. Do you want me to remove this extra blank space?

Reply:
Thank you for doing that, but it’s not part of an issue, that’s all.